internal/upload/koji/koji.go

```diff
@@ -492,3 +465,33 @@ func CreateKojiTransport(relaxTimeout uint) http.RoundTripper {
 
 	return rt
 }
+
+func CustomCheckRetry(ctx context.Context, resp *http.Response, err error) (bool, error) {
+	shouldRetry, retErr := rh.DefaultRetryPolicy(ctx, resp, err)
+
+	// DefaultRetryPolicy denies retrying for any certificate related error.
+	// Override it in case the error is a timeout.
+	if !shouldRetry && err != nil {
+		if v, ok := err.(*url.Error); ok {
+			if _, ok := v.Err.(x509.UnknownAuthorityError); ok {
+				// retry if it's a timeout
+				return strings.Contains(strings.ToLower(v.Error()), "timeout"), v
+			}
+		}
+	}
+
+	return shouldRetry, retErr
+}
+
+func CreateCustomRetryableClient() *rh.Client {
+	client := rh.NewClient()
+	client.Logger = rh.LeveledLogger(&LeveledLogrus{logrus.StandardLogger()})
+	client.CheckRetry = CustomCheckRetry
+	return client
+}
+
+func CreateRetryableTransport() *rh.RoundTripper {
```

Comment:
I think this and the previous function belongs in the next patch, no?

Also, the commit message seems to describe the next patch. This patch just drops log messages, right? Presumably because the previous one adds the important ones as `Info()` so these are not needed? I think a bit more of an explanation of the change in behaviour of the previous and this patch combined would be helpful.